ApollosProject/apollos-apps
# fix: Complete Daily Reading Interaction

## ✏️ Description

As is stands today, the `COMPLETE` action gets triggered on the Content Single either when the user lands on the screen, or when they press a "complete" button.

For Daily Habits, our consumption of Content Single interactions is going to be a crucial piece of giving user's their correct "status". In order to compensate for the complexity, we're standardizing our interactions so that EVERY Content Item will store an `OPEN` interaction on load.

On the API, if a church has autocomplete set to true, the followup to an `OPEN` interaction will be `COMPLETE`. If the church has `autocomplete` set to `false`, there will still be a "complete" button that will store the `COMPLETE` interaction. 

On the API, whenever the `COMPLETE` action is triggered, we will do a check to see if the Content Item is today's Reading. If it is, there will follow a new interaction: `COMPELTE_DAILY_HABITS`

This front-end PR prepares for those API changes

## 🔬 To Test
1. Load up the app using Chase Oaks
2. Navigate to the Daily Reading
3. The checkmark should STAY
4. You should see a record in the dB inside the interactions table that says `OPEN`, `COMPLETE` and `COMPLETE_DAILY_HABITS`

## 📸 Screenshots
n/a

## ⚠️ To-do Before Merge
- [ ] There will need to be a Cluster update to enable this that @camrun91 will be working on

## 📝 Checklist
